### DISCUSSION TITLE
Enabling https to apicurio registry

### DISCUSSION THREAD

**Discussion by githubing** [Apicurio/apicurio-registry#2843] [Q&A]
Hello, I am trying to enable https in cluster to apicurio registry and I have configured the deployment following the documentation https://www.apicur.io/registry/docs/apicurio-registry-operator/1.0.0/assembly-registry-maintenance.html#registry-https-in-cluster.
Apicurio restarts without problems but https seems not enabled. the port 8443 is not open and I don't get any info from logs. Do I need to do something else to enable https in apicurio or what I am missing please? thanks!

**Comment by jsenko** [comment]
Hello, 
- Can you show me what your k8s resources look like? Specifically the Deployment and Service (also Secret if you've generated it and are ok with publishing it). 
- Logs would be nice even if there is no interesting info (make sure you enabled debug logging: `spec.configuration.logLevel=DEBUG` in the CRD or LOG_LEVEL=DEBUG env. variable)
- Are you testing this from inside the cluster? The HTTPS connection will be available only via the service. If you want an external HTTPS connection, you need to setup Ingress/Route with HTTPS termination or re-encrypts it.
